gap/OscarInterface/PackageInfo.g

```diff
@@ -0,0 +1,99 @@
+#
+# OscarInterface: GAP interface to OSCAR
+#
+# This file contains package meta data. For additional information on
+# the meaning and correct usage of these fields, please consult the
+# manual of the "Example" package as well as the comments in its
+# PackageInfo.g file.
+#
+SetPackageInfo( rec(
+
+PackageName := "OscarInterface",
+Subtitle := "GAP interface to OSCAR",
+Version := "0.12.0-DEV",
+Date := "20/03/2023", # dd/mm/yyyy format
+License := "GPL-2.0-or-later",
+
+Persons := [
+  rec(
+    LastName := "Breuer",
+    FirstNames := "Thomas",
+    IsAuthor := true,
+    IsMaintainer := true,
+    Email := "sam@math.rwth-aachen.de",
+    WWWHome := "https://www.math.rwth-aachen.de/~Thomas.Breuer",
+    PostalAddress := Concatenation(
+               "Thomas Breuer\n",
+               "Lehrstuhl für Algebra und Zahlentheorie\n",
+                       "RWTH Aachen\n",
+               "Pontdriesch 14/16\n",
+               "52062 Aachen\n",
+               "Germany" ),
+    Place := "Aachen",
+    Institution := "RWTH Aachen",
+  ),
+  rec(
+    LastName := "Horn",
+    FirstNames := "Max",
+    IsAuthor := true,
+    IsMaintainer := true,
+    Email := "mhorn@rptu.de",
+    WWWHome := "https://www.quendi.de/math",
+    PostalAddress := Concatenation(
+               "Fachbereich Mathematik\n",
+               "RPTU Kaiserslautern-Landau\n",
+               "Gottlieb-Daimler-Straße 48\n",
+               "67663 Kaiserslautern\n",
+               "Germany" ),
+    Place := "Kaiserslautern, Germany",
+    Institution := "RPTU Kaiserslautern-Landau",
+  ),
+],
+
+SourceRepository := rec( Type := "git", URL := "https://github.com/oscar-system/Oscar.jl" ),
+IssueTrackerURL := "https://github.com/oscar-system/Oscar.jl/issues",
+PackageWWWHome := "https://www.oscar-system.org/",
+
+ArchiveURL     := Concatenation( ~.PackageWWWHome, "OscarInterface-", ~.Version ),
+PackageInfoURL := Concatenation( ~.PackageWWWHome, "PackageInfo.g" ),
+README_URL     := Concatenation( ~.PackageWWWHome, "README.md" ),
+
+ArchiveFormats := ".tar.gz",
+
+##  Status information. Currently the following cases are recognized:
+##    "accepted"      for successfully refereed packages
+##    "submitted"     for packages submitted for the refereeing
+##    "deposited"     for packages for which the GAP developers agreed
+##                    to distribute them with the core GAP system
+##    "dev"           for development versions of packages
+##    "other"         for all other packages
+##
+Status := "dev",
+
+AbstractHTML   :=  "",
+
+PackageDoc := rec(
+  BookName  := "OscarInterface",
+  ArchiveURLSubset := ["doc"],
+  HTMLStart := "doc/chap0_mj.html",
+  PDFFile   := "doc/manual.pdf",
+  SixFile   := "doc/manual.six",
+  LongTitle := "GAP interface to OSCAR",
+),
+
+Dependencies := rec(
+  GAP := ">= 4.12",
+  NeededOtherPackages := [ ],
```

Comment:
Logically, JuliaInterface will be a needed package.
In practice, it is clear that JuliaInterface will be available when OscarInterface gets loaded.
I am not sure whether JuliaInterface should be listed here.

Reply:
I'll add it